test/utils/restart_driver_daemonset.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+# Copyright 2021 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -euo pipefail
+
+echo "Deleting the Driver node daemonset ..."
+kubectl delete -f ./deploy/csi-azurefile-node.yaml
```

Comment:
```console
kubectl delete -f ./deploy/csi-azurefile-node.yaml --ignore-not-found
kubectl delete -f ./deploy/csi-azurefile-node-windows.yaml --ignore-not-found
```

Reply:
are you waiting for a few seconds after delete and then after install daemonset, e.g. `sleep 15`

Reply:
Yes, I have added `sleep 15`. It's fixed now.